x-pack/plugins/task_manager/server/monitoring/capacity_estimation.ts

```diff
@@ -80,9 +80,10 @@ export function estimateCapacity(
   /**
    * Given the current configuration how much task capacity do we have?
    */
-  const capacityPerMinutePerKibana = Math.round(
-    ((60 * 1000) / (averagePollIntervalsPerExecution * pollInterval)) * maxWorkers
-  );
+  const capacity = (60 * 1000) / (averagePollIntervalsPerExecution * pollInterval);
```

Comment:
`capacityPerMinutePerKibana` is based on the current configuration so this value will always be the same for the default configuration of 10 workers and 3 second poll interval. The value that is an estimate is `assumedKibanaInstances` which is used in the calculation for `assumedAverageRecurringRequiredThroughputPerMinutePerKibana` which is then used in `getHealthStatus`. I think instead of having an upper and lower bound for capacity per minute per kibana, we should be doing something like:

```
const assumedKibanaInstances = Math.max(workload.owner_ids, 1);
const assumedKibanaInstancesLower = assumedKibanaInstances - 1 (shouldn't go below 1 tho)
const assumedKibanaInstancesUpper = assumedKibanaInstances + 1
```

and then calculate a lower & upper `assumedAverageRecurringRequiredThroughputPerMinutePerKibana` to pass to getHealthStatus. Then compare `capacityPerMinutePerKibana` to these bounds to determine health status and the warning message. I haven't thought through what the comparisons should look like but if you want to talk through it, let me know!

Reply:
okay thanks! this make sense